R/data.R

```diff
@@ -0,0 +1,17 @@
+#' Antidepressant trial data.
+#'
+#' A dataset containing the data from a public available antidepressant clinical trial of an active drug versus placebo.
+#' The relevant endpoint is the Hamilton 17-item rating scale for depression (HAMD17) which was assessed at baseline and weeks 1, 2, 4, and 6.
+#' Study drug discontinuation occurred in 24% (20/84) for the active drug and 26% (23/88) for placebo.
```

Comment:
These numbers seem to lack context  what is the 84 and 88 ?

Reply:
they were the absolute number of patients, but actually this information is not necessary